object/ldap_conn.go

```diff
@@ -235,6 +240,8 @@ func SyncSingleUser(
 			return errors.New("LDAP sync error: failed to AddUser")
 		}
 
+		localUserID = newUser.Id
```

Comment:
Explicit is better than implicit. Please return ID from AddUser method.

If you dont want to fix AddUser() usages all across the repo, rename it to AddUserOld and make a copy of it with the name AddUser, returning ID.

Reply:
done